Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:10.7.0
+
+# install npm dependencies
+WORKDIR /app
+COPY ./package.json /app/package.json
+RUN npm install
+
+# copy the code
+# (after dependencies installation)
+COPY . /app
+
+# run the application
+EXPOSE 1969
+ENTRYPOINT [ "./docker-entrypoint.sh" ]
```

Comment:
I think you want here instead start the app with `npm start`. The Dockerfile here overall could try to follow the structure of https://github.com/nodejs/docker-node/blob/72dd945d29dee5afa73956ebc971bf3a472442f7/8/onbuild/Dockerfile (ignore the ONBUILD and ENV stuff there).

Reply:
`npm start` won't currently [pull translators automatically](https://github.com/zotero/translation-server-v2/issues/1), which the entrypoint script does.

Reply:
Ah, I see. And `npm start` is part of the other script. Okay, then forget my comment here.